testing/refpolicy/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=refpolicy
+pkgver=20170204
+pkgrel=0
+pkgdesc="SELinux policy reference"
+url="https://github.com/TresysTechnology/refpolicy/wiki"
+arch="noarch"
+license="GPLv2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev checkpolicy python gawk"
+install=""
+subpackages="$pkgname-doc"
+source="https://raw.githubusercontent.com/wiki/TresysTechnology/refpolicy/files/refpolicy-2.$pkgver.tar.bz2"
+builddir="$srcdir/refpolicy"
+
+# refpolicy config
+monolithic=n
+# This causes various distro-specific policies to be built. Since most SELinux
+# policies are written against redhat-based distros, we set that here.
+distro=redhat
+unknown_perms=deny
+
+# These are somewhat related to what is in the CentOS spec file, although they
+# are slightly differnet in what they install.
+makeCmds() {
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=1024 bare || return 1
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=1024 conf || return 1
+}
+
+installCmds() {
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=0124 SEMOD_EXP="/usr/bin/semodule_expand -a" base.pp
+	make validate UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=1024 SEMOD_EXP="/usr/bin/semodule_expand -a" modules
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=0124 DESTDIR="$pkgdir" install
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=0124 DESTDIR="$pkgdir" install-appconfig
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=0124 DESTDIR="$pkgdir" install-docs
+	make UNK_PERMS=$3 NAME=$1 TYPE=$2 DISTRO=$distro UBAC=n DIRECT_INITRC=y MONOLITHIC=$monolithic MLS_CATS=1024 MCS_CATS=0124 DESTDIR="$pkgdir" install-headers
+}
+
+build() {
+	cd "$builddir"
+	makeCmds targeted mcs $unknown_perms || return 1
+}
+
+package() {
+	cd "$builddir"
+	echo packaging now
+	installCmds targeted mcs $unknown_perms || return 1
+	mkdir -p $pkgdir/usr/share/selinux/devel || return 1
+	mv "$pkgdir/usr/share/selinux/targeted/include" "$pkgdir/usr/share/selinux/devel/include"
```

Comment:
It sort of depends, in ubuntu/debian/rhel, there are several packages all built with various policies enabled and disabled, but I think they all install to mostly the same target directories. I only built one set (the devel set) in this package for now, so that's why I did a `mv`. I can make it a `cp -r`, though I suspect this will need to change back when someone starts using this in anger.